ideas/225-status-index.md

```diff
@@ -0,0 +1,130 @@
+future-iteration: no
+roles-needed:
+    - Dev
+    - Designer
+    - Community
+okrs:
+    -
+    -
+---
+
+## Preamble
+
+    Idea: 000-status-index
+    Title: Status Index
+    Status: draft
+    Created: 2018-04-27
+
+## Summary
+
+We have a growing number of very useful standalone webpages that are providing valuable
+informations and services. 
+
+Examples of those are:
+- cluster visualization,
+- cluster status,
+- history of the #status channel on the different whisper networks,
+- nightly build QR codes page (https://status-im.github.io/nightly/)
+- status.im people list (https://rawgit.com/jacqueswww/b5460887fe86871434dcef833fa00c92/raw/status_people.html)
+- documentation
+
+The goal of this swarm is to build an index of these pages so that they can be listed
+and easily accessed from a single place. We also want to make sure that each of these
+pages are documented to make it easy to reproduce them or for someone other than their
+creator to contribute.
+
+## Current status
+
+- weekly sync calls tbd
+
+## Swarm Participants
+
+- Lead Contributor: yenda
+- Evaluator (defaults to lead contributor):
+- Contributor: 
+- Contributor:
+- QA:
+- PM (required for user-facing):
+- UX(R) (required for user-facing swarms):
+
+## Product Overview
+
+The purpose of this Swarm is to build an index of all the standalone webpages that 
+have been built by various members of the team and should be made accessible via a 
+simple single directory page. We will evaluate cloud/container provider to host/run 
+those websites as well.
+
+It also aims at pushing production of documentation to make it easier to contribute to
+this projects and troubleshoot them.
+
+### User Stories
+
+> What user stories are you solving?
+As as status core developper, I have troubles with messaging, I suspect the cluster
+to be responsible, I want to be able to go to index.status.im and find the page that
+will show me the current status of the cluster.
+
+As a status community member, I want to create a bot that interacts with status clients
+with status communication protocol. I want to be able to easily find the documentation 
+of the communication protocol by checking index.status.im
+
+As a status community member, I want to improve the static website rendering status
+ideas repository. I want to be able to check on index.status.im and find the documentation
+on how this website is built so that I can contribute.
+
+### Security and Privacy Implications
+
+None
+
+## Dates
+
+Weekly sync call tbd
+
+### Minimum Viable Product
+
+> Mandatory, completes the Idea in the fastest route possible, can be hacky,
+> needed to feel progress. See https://imgur.com/a/HVlw3
+
+Goal Date:
+
+Description:
+
+### Iteration 1
+
+Goal Date:
+
+Description:
+
+## Success Metrics
+
+> Assuming the idea ships, what would success look like? What are the most
+> important metrics that you would move?
+>
+> Example: Onboarding conversion rate. Target >30% full funnel
+
+## Exit criteria
```

Comment:
It'd be useful if we can figure out exit criteria and success metrics, this way the problem/swarm is bounded in terms of what the most important thing it is trying to do is. Also to figure out the overlap with existing documentation efforts, etc.